packages/data/addon/models/navi-dimension.ts

```diff
@@ -32,6 +34,16 @@ export default class NaviDimensionModel extends NativeWithCreate implements Navi
    */
   declare suggestions?: Readonly<Record<string, string>>;
 
+  /**
+   * Manual Entry Input
+   */
+  declare manualInputEntry?: Readonly<unknown>;
+
+  /**
+   * Manual Entry Input Query
+   */
+  declare manualQuery?: Readonly<unknown>;
```

Comment:
This is a general dimension model. We probably shouldn't put use-case specific fields here.